utils/evaluate_utils.py

```diff
@@ -81,7 +83,7 @@ def class_PR_curves(
         axs[ax_x, ax_y].set(xlabel="Recall", ylabel="Precision")
 
         ax_x += 1
-        if ax_x == 4:
+        if ax_x == 3:
```

Comment:
Consider adding some comments here and around this line to describe what's happening.

Reply:
Added comments in https://github.com/WayScience/phenotypic_profiling_model/pull/21/commits/7a024fd3f9feb6e0e3a69296cea221d48c5e52c0